plenoptic/synthesize/synthesis.py

```diff
@@ -1075,117 +972,89 @@ def save(self, file_path, save_model_reduced=False, attrs=['model'],
             save_dict[k] = attr
         torch.save(save_dict, file_path, pickle_module=dill)
 
-    @classmethod
     @abc.abstractmethod
-    def load(cls, file_path, model_attr_name='model', model_constructor=None, map_location='cpu',
-             **state_dict_kwargs):
-        r"""load all relevant stuff from a .pt file
+    def load(self, file_path, map_location='cpu',
+             check_attributes=['base_signal', 'base_representation',
```

Comment:
Should this even be optional?

Reply:
Should `check_attributes` be optional? See my response in your next comment about this.